### PR TITLE
ci: Remove badges from README.md prior to converting to HTML

### PR DESCRIPTION
**Pull request by spetrosi**
- Remove thematic break after badges
- Remove badges from README.md prior to converting to HTML

Signed-off-by: Sergei Petrosian <spetrosi@redhat.com>

### CODE REVIEW

**Review by liangwen12year — COMMENTED** [review]
Maybe explain the reason that necessitates this commit ?

File: .github/workflows/markdownlint.yml [hunk 1]
```diff
@@ -34,5 +34,5 @@ jobs:
         with:
           args: >-
             --ignore=CHANGELOG.md
-            **/*.md 
+            **/*.md
```
Comment (liangwen12year) [inline .github/workflows/markdownlint.yml:37]: This should be in another commit ?

Comment (spetrosi) [reply .github/workflows/markdownlint.yml:37]: I am doing it via automation, so pushing all in a single commit. That's a minor change, last time I made a bug that kept this extra space, fixing it here. I believe it's fine to have it within the fixing commit. This should be the last PR for badges, I made a lot of noise already anyway...
